### PR TITLE
Implement lic_2

### LINKED ISSUE #6: Implement function for LIC 2
Implement a function that checks LIC 2 and add unit tests as well. This function should be called when creating the CMV. 
From the specification: 
```
2. There exists at least one set of three consecutive data points which form an angle such that:
angle < (PI−EPSILON)
or
angle > (PI+EPSILON)
The second of the three consecutive points is always the vertex of the angle. If either the first
point or the last point (or both) coincides with the vertex, the angle is undefined and the LIC
is not satisfied by those three points.
(0 ≤ EPSILON < PI)
```

### PR DESCRIPTION
**Pull request by Penguinification**
This adds an implementation for the lic_2 function. It returns true when the angle satisfies angle > (PI-EPSILON) (or equivalently when the outer angle satisfies angle < (PI+EPSILON)).

This also adds a set of unit tests. There is a testcase for when the condition is true and another for when it is false. There are also 4 test cases that handle the edge case when the first or the last points in a sequence coincide with the vertex.

Fixes #6

### CODE REVIEW

File: test_cmv.py [hunk 1]
```diff
@@ -59,6 +59,81 @@ def test_lic_0_false():
...
+    assert(not result)
+
+def test_lic_2_coinciding_vertex_1():
+    """
+    Test that lic_2 returns false when the first point in one sequence
+    coincides with the vertex and no other sequence of consecutive points
+    satisfies angle < (PI-EPSILON)
+    """
+    parameters = {
+        "epsilon": pi
```
Comment (editf) [inline test_cmv.py:92]: epsilon should be < pi as defined in the specification (same on line 118), apart from that it looks good!

**Review by editf — APPROVED** [review]

**Comment by Penguinification** [comment]
Oh wait, maybe I shouldn't have merged yet. I'll open an issue to fix the test case.
